packages/core/src/components/Toggle/Toggle.tsx

```diff
@@ -1,14 +1,26 @@
 import { WithStyle } from '@medly-components/utils';
-import type { FC } from 'react';
-import { forwardRef, memo, useCallback, useMemo } from 'react';
+import { cloneElement, FC, forwardRef, memo, useCallback, useMemo } from 'react';
 import FieldWithLabel from '../FieldWithLabel';
 import * as Styled from './Toggle.styled';
 import { ToggleProps } from './types';
 
 const Component: FC<ToggleProps> = memo(
     forwardRef((props, ref) => {
-        const { id, size, label, required, labelPosition, labelVariant, labelWeight, labelColor, fullWidth, onChange, ...restProps } =
-            props;
+        const {
+            id,
+            size,
+            label,
+            required,
+            labelPosition,
+            labelVariant,
+            labelWeight,
+            labelColor,
+            fullWidth,
+            onChange,
+            iconLeft,
+            iconRight,
```

Comment:
Can we use `OnIcon` and `OffIcon` to be more clear? Also this is the naming convention in the styled file, we can keep it consistent here